app/src/main/res/values/games-ids.xml

```diff
@@ -6,11 +6,17 @@ Save this file as res/values/games-ids.xml in your project.
 <resources>
     <string name="app_id" translatable="false">429371117063</string>
     <string name="achievement_signin_palooza" translatable="false">CgkIh5yNxL8MEAIQAA</string>
-    <string name="achievement_cinephile" translatable="false">CgkIh5yNxL8MEAIQAQ</string>
     <string name="achievement_gdg_maniac" translatable="false">CgkIh5yNxL8MEAIQAg</string>
     <string name="achievement_thanks_for_coming_back." translatable="false">CgkIh5yNxL8MEAIQAw</string>
     <string name="achievement_king_of_the_hill" translatable="false">CgkIh5yNxL8MEAIQBA</string>
+    <string name="achievement_looking_for_experts" translatable="false">CgkIh5yNxL8MEAIQDw</string>
```

Comment:
Hey man :smile: the PR is originated from `PareshMayani:develop` which I don't have access. 
You somehow pulled my commit from branch `refactor/358-achievements` into your develop branch. You should remove it.